OptionsDialog.cpp

```diff
@@ -232,10 +235,11 @@ void OptionsDialog::LoadProfileInfo(std::function<void()> onFinished) {
         if (d.IsObject() && d.HasMember("items") && d["items"].IsArray() && d["items"].Size() > 0 && d["items"][0].HasMember("contentDetails")) {
             const rapidjson::Value &i = d["items"][0]["contentDetails"]["relatedPlaylists"];
 
-            m_userPlaylists.push_back({ Tools::ToWString(i["favorites"]), Plugin::instance()->Lang(L"YouTube.Playlists\\Favorites"), true, L"Favorites" });
-            m_userPlaylists.push_back({ Tools::ToWString(i["uploads"]), Plugin::instance()->Lang(L"YouTube.Playlists\\Uploads"), false, L"Uploads" });
-            m_userPlaylists.push_back({ Tools::ToWString(i["likes"]), Plugin::instance()->Lang(L"YouTube.Playlists\\Likes"), true, L"Likes" });
-            m_userPlaylists.push_back({ Tools::ToWString(i["watchLater"]), Plugin::instance()->Lang(L"YouTube.Playlists\\WatchLater"), true, L"WatchLater" });
+            // YouTube seems to no longer provide favorites or watchLater

```

Comment:
Hmm, the comment says that `favorites` or `watchLater` may not be present, but the code checks for `favorites` and `uploads`?

Reply:
I made them all optional encase Google changes the api further. If there was a common ID I would provide it. There was no common ID for favorites.

Reply:
but `watchLater` and `likes` are not checked

Reply:
For watchLater and likes they have shorthand ids, so I use a ternary there. I'd imagine they should still exist, even if not in relatedPlaylists.

Reply:
oh right, I see it now